x/evm/genesis_test.go

```diff
@@ -83,40 +81,43 @@ func TestInitGenesis(t *testing.T) {
 			},
 			expPanic: true,
 		},
-		{
-			name: "invalid account type",
-			malleate: func(network *testnetwork.UnitTestNetwork) {
-				acc := authtypes.NewBaseAccountWithAddress(address.Bytes())
-				network.App.AccountKeeper.SetAccount(network.GetContext(), acc)
-			},
-			genState: &types.GenesisState{
-				Params: types.DefaultParams(),
-				Accounts: []types.GenesisAccount{
-					{
-						Address: address.String(),
-					},
-				},
-			},
-			expPanic: true,
-		},
-		{
-			name: "invalid code hash",
-			malleate: func(network *testnetwork.UnitTestNetwork) {
-				ctx := network.GetContext()
-				acc := network.App.AccountKeeper.NewAccountWithAddress(ctx, address.Bytes())
-				network.App.AccountKeeper.SetAccount(ctx, acc)
-			},
-			genState: &types.GenesisState{
-				Params: types.DefaultParams(),
-				Accounts: []types.GenesisAccount{
-					{
-						Address: address.String(),
-						Code:    "ffffffff",
-					},
-				},
-			},
-			expPanic: true,
-		},
+		// TODO: similar to the other tests below this can be removed, check below
+		//{
+		//	name: "invalid account type - missing code",
+		//	malleate: func(network *testnetwork.UnitTestNetwork) {
+		//		acc := authtypes.NewBaseAccountWithAddress(address.Bytes())
+		//		network.App.AccountKeeper.SetAccount(network.GetContext(), acc)
+		//	},
+		//	genState: &types.GenesisState{
+		//		Params: types.DefaultParams(),
+		//		Accounts: []types.GenesisAccount{
+		//			{
+		//				Address: address.String(),
+		//			},
+		//		},
+		//	},
+		//	expPanic: true,
+		//},
+		// TODO: similar to the other test below this can be removed because the integrity of code hashes between
+		// account keeper and EVM state is not checked anymore.
+		//{
+		//	name: "invalid code hash",
+		//	malleate: func(network *testnetwork.UnitTestNetwork) {
+		//		ctx := network.GetContext()
+		//		acc := network.App.AccountKeeper.NewAccountWithAddress(ctx, address.Bytes())
+		//		network.App.AccountKeeper.SetAccount(ctx, acc)
+		//	},
+		//	genState: &types.GenesisState{
+		//		Params: types.DefaultParams(),
+		//		Accounts: []types.GenesisAccount{
+		//			{
+		//				Address: address.String(),
+		//				Code:    "ffffffff",
+		//			},
+		//		},
+		//	},
+		//	expPanic: true,
+		//},
```

Comment:
I think all of these tests can be removed because the cross-checking with the auth module is not necessary anymore 👀